cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.example

```diff
@@ -0,0 +1,126 @@
+## All options defined here are available to all instances.
+#
+init_config:
+
+  ## @param min_collection_interval - int - optional - default:10
+  ## Active session sampling interval in seconds
+  # min_collection_interval: <interval_in_seconds>
+
+## Every instance is scheduled independent of the others.
+#
+instances:
```

Comment:
which one of these is the config to actually enable activity collection?

Reply:
Activity sampling is enabled with setting `dbm: true`. Currently, there's no dedicated parameter for enabling/disabling activity sampling. I see no point in disabling activity samples, as every other DBM feature depends on it.

Reply:
sure, but the way all of our other integrations work is they allow for granular control over every single feature. Maybe this is fine for now, but when we add query metrics support fully we should allow for customers to turn on/off activity if they want